src/components/cc-global-header.vue

```diff
@@ -128,11 +128,42 @@ export default defineComponent({
       ];
     } else {
       vm.menuLoading = true;
+      axios.interceptors.response.use(
+        undefined,
+        function retryApiRequest(error) {
+          var config = error.config;
+          // If config does not exist or the retry option is not set, reject
+          if (!config || !config.retry) return Promise.reject(error);
+
+          // Set the variable for keeping track of the retry count
+          config.retryCount = config.retryCount || 0;
+
+          // Check if we've maxed out the total number of retries
+          if (config.retryCount >= config.retry) {
+            // Reject with the errors
+            return Promise.reject(error);
+          }
+
+          // Increase the retry count
+          config.retryCount += 1;
+
+          // Create new promise for exponential backoff
+          var exponentialBackOff = new Promise(function (resolve) {
```

Comment:
If I understand correctly, we're using a fixed retry delay. If that is the case, we don't need to call this "exponential backoff" since it might lead to confusion.
```suggestion
          var retryDelay = new Promise(function (resolve) {
```